lib/auxiliary/time_helper.rb

```diff
@@ -24,5 +24,9 @@ def seconds_to_time(sec)
     def time_diff(start, finish)
       ((finish - start) * 24 * 60 * 60).to_i
     end
+
+    def add_timestamp
```

Comment:
This function does not add a timestamp it gets a timestamp. As `get_timestamp` is a bad name from a Ruby point of view, use `create_timestamp`, `generate_timestamp`, `current_timestamp`, etc